src/plugins/home/public/application/components/solutions_section/solution_title.test.tsx

```diff
@@ -42,121 +43,51 @@ const solutionEntry = {
   order: 1,
 };
 
+const mockTitle = 'Page Title';
+const makeProps = () => ({
```

Comment:
Why is this a closure? Does this data change at some point?

Reply:
It is meant to prevent a situations where the props are modified by any of the tests.